metrics/web/test_driver/integration_test.dart

```diff
@@ -0,0 +1,172 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:metrics/auth/presentation/strings/auth_strings.dart';
+import 'package:metrics/auth/presentation/widgets/auth_form.dart';
+import 'package:metrics/common/presentation/button/widgets/metrics_positive_button.dart';
+import 'package:metrics/common/presentation/strings/common_strings.dart';
+import 'package:metrics/common/presentation/widgets/metrics_text_form_field.dart';
+import 'package:metrics/dashboard/presentation/pages/dashboard_page.dart';
+import 'package:metrics/dashboard/presentation/strings/dashboard_strings.dart';
+import 'package:metrics/dashboard/presentation/widgets/build_number_scorecard.dart';
+import 'package:metrics/dashboard/presentation/widgets/build_result_bar_graph.dart';
+import 'package:metrics/dashboard/presentation/widgets/coverage_circle_percentage.dart';
+import 'package:metrics/dashboard/presentation/widgets/performance_sparkline_graph.dart';
+import 'package:metrics/dashboard/presentation/widgets/project_metrics_tile.dart';
+import 'package:metrics/dashboard/presentation/widgets/projects_search_input.dart';
+import 'package:metrics/main.dart';
+import 'package:metrics/project_groups/presentation/widgets/add_project_group_card.dart';
+import 'package:universal_io/io.dart';
+
+import 'arguments/model/user_credentials.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group("LoginPage", () {
+    testWidgets("shows an authentication form", (WidgetTester tester) async {
+      await _pumpApp(tester);
+
+      expect(find.byType(AuthForm), findsOneWidget);
+    });
+
+    testWidgets("can authenticate in the app using an email and a password",
+        (WidgetTester tester) async {
+      await _pumpApp(tester);
+
+      await _login(tester);
+
+      expect(find.byType(DashboardPage), findsOneWidget);
+    });
+
+    testWidgets("can log out from the app", (WidgetTester tester) async {
+      await _pumpApp(tester);
+
+      await _openUserMenu(tester);
+
+      await tester.tap(find.text(CommonStrings.logOut));
+      await tester.pumpAndSettle(const Duration(seconds: 2));
+
+      expect(find.byType(AuthForm), findsOneWidget);
+    });
+  });
+
+  group("DashboardPage", () {
+    testWidgets("loads projects and shows the project tiles",
+        (WidgetTester tester) async {
+      await _pumpApp(tester);
+
+      await _login(tester);
+
+      expect(find.byType(ProjectMetricsTile), findsWidgets);
+    });
```

Comment:
```suggestion
    },);
```
Please, use the trailing comma in such cases.